### ISSUE TITLE
No examples of what sorts of queries can be performed.

### ISSUE THREAD

**Issue by mrshirts** [choderalab/thermopyl#26]

**Comment by jchodera** [comment]
Basic example added in #27. Will try to add more once I figure them out, but would be great if Bryce could add example code as he digs up density and enthalpy data.  @mrshirts : Can you tag his github handle?

**Comment by mrshirts** [comment]
@manubay: for the record.

**Comment by mrshirts** [comment]
Whoops.  @bmanubay

**Comment by bmanubay** [comment]
@jchodera 

Hi John,

Just for the sake of consistency with how you defined the pandas data frame variable in the example provided, the last line of code in the "Use Pandas to query the experimental literature" should read:

`# Extract rows with mass densities
densities = df[np.isnan(df['Mass density, kg/m3'])==False]
# Get a list of unique components
unique_components = set(df['components'])` 

Correct?

**Comment by jchodera** [comment]
Yes, sorry!  Can you submit a pull request to correct the `README.md`?

**Comment by bmanubay** [comment]
Sure!
